alpine/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG FROM_IMAGE=python:3-alpine3.7
+ARG FROM_IMAGE=python:alpine
```

Comment:
Will this potentially get us a newer version of Alpine? Could that cause issues?

Reply:
Yes. It's unlikely to cause issues, but it could theoretically. But basically these images are never built without specifying the argument so it's low-risk.